src/models/Diary.js

```diff
@@ -19,7 +19,8 @@ const DiarySchema = new Schema({
     required: true,
   },
   publish_date: {
-    type: String,
+    type: Date, 
```

Comment:
Is there a reason to store `publish_date` as type `Date`? 
This is what the doc says: `publish_date - a string in ISO 8601 format representing the date the entry was published`